docs/administration/eboard-onboarding-offboarding.rst

```diff
@@ -18,82 +22,101 @@ Follow these steps to execute an election.
+    - Open call for nominations: club members nominate others in community for eboard positions
   
     - Verify / confirm nominations with nominees
   
     - Accept nominations in a Google Drive form
 
 - **Week 12**: Hold elections during club meeting
 
-    - At start of Week 12, open election with all nominated candidates
-    
-    - Accept votes in a Google Drive form that requires logins to vote (to prevent voter fraud); remind club members that voting is anonymous
+    - At start of Week 12 (Monday), open election with nominated candidates
     
-    - After accepting votes, close form, tally responses
+    - Create CampusGroups survey to accept votes (see :doc:`campusgroups-management` for more info); remind club members that voting is anonymous
     
-    - At Week 12 meeting, announce results to club members in room; follow-up with an email announcment after
-    
-    - After announcing winners, current eboard should **permanently delete** the data associated with voting form (to protect voter identity)
+    - Close form at end of Week 12 (Sunday)
 
-Don't forget to congratulate all nominees for their participation in the election.
+    - Current eboard members reach out to winning members to confirm their acceptance of role
+    
+    - At Week 13 meeting, announce results to club members in room; follow-up with an email announcment after
 
+Congratulate all nominees for their participation in the election.
 
 CampusGroups roles
-^^^^^^^^^^^^^^^^^^
+==================
 
-All RIT clubs are required to use CampusGroups and maintain records there for the clubs office.
+RIT clubs are required to use CampusGroups and maintain records for the clubs office.
 After holding an election, update the incoming eboard members inside of CampusGroups.
 
 Detailed documentation on CampusGroups can be found on the :doc:`campusgroups-management` page.
 
-
 Introduce faculty adviser
-^^^^^^^^^^^^^^^^^^^^^^^^^
+=========================
 
-Outgoing eboard members should help incoming members maintain a relationship with the faculty adviser.
+Outgoing eboard members help incoming members maintain a relationship with the faculty adviser.
 Arrange a meeting time for the outgoing and incoming eboard members to meet with the faculty adviser.
 
-See more information on the :doc:`faculty-adviser` page.
+.. seealso::
 
+   See :doc:`faculty-adviser` for more information.
 
 Other on-boarding tasks
-^^^^^^^^^^^^^^^^^^^^^^^
+=======================
 
-- Grant operator privileges to incoming eboard members in the IRC channel (see how in :doc:`../infrastructure/irc-channel`).
+- Share Runbook with incoming eboard members, encourage questions and feedback on improving the Runbook
 
-- Add incoming eboard members as admins to `Facebook group`_
+- Incoming eboard members must take `financial certification test`_ from RIT clubs office (all eboard members **must be certified** to use club funds)
+
+- Add incoming eboard to private Slack channel for eboard discussion
```

Comment:
Shouldn't they also be given admin? As far as I understand, that's how we're doing things now.

Reply:
@ct-martin Looking at how we do it now, that is how it's set up. I pushed a new commit to add an additional step.